src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -431,6 +429,7 @@ <h2 class="tw-whitespace-pre-wrap tw-text-oebblack tw-font-bold">
 									class="tw-flex tw-flex-col md:tw-flex-row md:tw-justify-between tw-gap-5 md:tw-gap-0"
 								>
 									<oeb-input
+										id="{{ 'competencyTitle_' + i }}"
```

Comment:
I feel like multiple tickets are being mixed up here. Or what is the purpose of this change?

Reply:
these issues were caused because the main branch was already ahead again by Youssef's change with this commit baca52e  (because you're right this has nothing to do with this ticket 👍 )

I updated the branch since